### ISSUE TITLE
Getting “Commercial break on progress” on latest ipa

### ISSUE THREAD

**Issue by Sukixo** [level3tjg/TwitchAdBlock#25]
Installed the latest .ipa and receiving the “commercial break in progress” screen when clicking on a stream.

**Comment by level3tjg** [comment]
Did you have low latency player enabled in the player settings?

**Comment by Sukixo** [comment]
> Did you have low latency player enabled in the player settings?

yes, that specific streamer isn’t live atm but i’ll turn it off and report back when they’re live.

**Comment by Sukixo** [comment]
> Did you have low latency player enabled in the player settings?

Still get the commercial break in progress

**Comment by level3tjg** [comment]
Some app changes have caused client side ad blocking to stop working. For the record you should keep low latency player on, the code for blocking server side ads will never run if it's off.

**Comment by Sukixo** [comment]
> Some app changes have caused client side ad blocking to stop working. For the record you should keep low latency player on, the code for blocking server side ads will never run if it's off.

okay, tried downgrading just to see if that would work and it still gives the commercial break, every single stream i’ve tried works perfectly fine except that one stream. Just to note before you updated i never had gotten the commercial break screen before.
